smarts/core/road_map.py

```diff
@@ -509,3 +530,51 @@ def signed_lateral_error(self, p) -> float:
     def dist_to(self, p) -> float:
         """Calculates straight line distance to the given 2D point"""
         return np.linalg.norm(self.pos - p[: len(self.pos)])
+
+
+class WaypointsCache:
```

Comment:
Did this get moved?  It's still showing up in `RoadMap` for me.

Reply:
Sorry, it was moved but i forgot to remove it from `road_map.py`.